scripts/get_dbdeployer_mysql_json.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
```

Comment:
To comply with the sanity check,
* Line 2 should contain the word `DBdeployer`
* Line 3 should contain the word `Copyright` with the author attribution